budcamp/Homework3/Homework3/task1/Poland.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework3
+{
+    internal class Poland: History, ICountry
```

Comment:
Same here, `Poland` **has a** `History` makes much more sense then `Poland` **is a** `History`